quinn-proto/src/crypto/rustls.rs

```diff
@@ -219,6 +220,19 @@ impl crypto::Session for TlsSession {
         let (aad, tag) = pseudo_packet.split_at_mut(tag_start);
         key.open_in_place(nonce, aead::Aad::from(aad), tag).is_ok()
     }
+
+    fn export_keying_material(
+        &self,
+        output: &mut [u8],
+        label: &[u8],
+        context: Option<&[u8]>,
```

Comment:
```suggestion
        context: &[u8],
```

As far as I can tell there's no special meaning to `None` here; [rustls just converts it to an empty slice](https://github.com/ctz/rustls/blob/main/rustls/src/key_schedule.rs#L357). Unless there's some subtlety I'm missing, we should keep our signature simple.

Reply:
Hmm, good point.

Digging into the RFCs, it turns out that in TLS1.2 and before, 'no context' and 'empty context' were different, but in TLS1.3 they are the same. This explains the rustls type signature, and seems to mean that we're fine getting rid of the Option.

(ref: Section 7.5 of RFC 8446)